tests/test_lora_variants.py

```diff
@@ -80,6 +104,16 @@ def forward(self, input_ids, dummy_image_input):
 ]
 
 
+CONV_GROUPS_TEST_CASES = [
```

Comment:
I think for this case, we don't need to assign `CONV_GROUPS_TEST_CASES`, just put the settings below directly inside of `pytest.mark.parametrize`.